go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/traefik/mocktail
 
-go 1.19
+go 1.22
```

Comment:
```suggestion
go 1.21
```

Reply:
Why are you suggesting to set go 1.21 in `go.mod` while keeping go `1.22` in CI :thinking:  ?